kleva-j/iReporter
# Feature/#162527254/page list of resolved incident records

#### What does this PR do?
- Adds a page to view the list of resolved incident records

#### Description of the task to be completed
- create page to view resolved red-flag records
- create page to view resolved intervention records
- Create a new user using a database.

#### How should this be manually tested?
- Clone the repository
- Check out into the branch Feature/#162527254/page-list-of-resolved-incident-records
- Install all dependencies
- run `npm start` to run the app successfully.

#### What are the relevant pivotal tracker stories?
[162527254](https://www.pivotaltracker.com/story/show/162527254)
